central/complianceoperator/v2/suites/datastore/datastore.go

```diff
@@ -0,0 +1,52 @@
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	pgStore "github.com/stackrox/rox/central/complianceoperator/v2/suites/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+)
+
+// DataStore is the entry point for storing/retrieving compliance operator suite.
+//
+//go:generate mockgen-wrapper
+type DataStore interface {
+	// GetSuite returns the suite for the given id
+	GetSuite(ctx context.Context, id string) (*storage.ComplianceOperatorSuite, bool, error)
+
+	// GetSuitesByCluster retrieve suites by the cluster
+	GetSuitesByCluster(ctx context.Context, clusterID string) ([]*storage.ComplianceOperatorSuite, error)
+
+	// UpsertSuite adds the suite to the database
+	UpsertSuite(ctx context.Context, suite *storage.ComplianceOperatorSuite) error
+
+	// UpsertSuites adds the suites to the database
+	UpsertSuites(ctx context.Context, suites []*storage.ComplianceOperatorSuite) error
+
+	// DeleteSuite removes a suite from the database
+	DeleteSuite(ctx context.Context, id string) error
+}
+
+// New returns an instance of DataStore.
+func New(complianceSuiteStorage pgStore.Store) DataStore {
+	ds := &datastoreImpl{
+		store: complianceSuiteStorage,
+	}
+	return ds
+}
+
+// NewForTestOnly returns an instance of DataStore only for tests.
+func NewForTestOnly(_ *testing.T, complianceSuiteStorage pgStore.Store) DataStore {
```

Comment:
nit: I could be wrong, but I don't think this is being used and could be delted.